### PR TITLE
Add Codestral example in JS / TS

### PR DESCRIPTION
**Pull request by tizkovatereza**
I added an equivalent example for the Codestral in Python, but in Typescript.

### CODE REVIEW

File: examples/codestral-code-interpreter-ts/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+First, cone the E2B Cookbook repository, and navigate to the correct directory.
+
+
+## Install dependencies
+
+Ensure all dependencies are installed via `npm install`.
+
+- `@mistralai/mistralai`
+- `@e2b/code-interpreter`
+- `dotenv`
```
Comment (mlejva) [inline examples/codestral-code-interpreter-ts/README.md:17]: Remove this:
1. it's not all dependencies used anyway
2. No reason to enumerate deps

File: examples/codestral-code-interpreter-ts/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Create a `.env` file in the project root directory and add your API keys:
+
+- Copy `.env.template`   to `.env`
+- Get the [E2B API KEY](https://e2b.dev/docs/getting-started/api-key)
+- Get the [MISTRAL API KEY](https://console.mistral.ai/api-keys/)
+
+# Usage
+
+Ensure you have the `city_temperature.csv` dataset file in the project root directory.
```
Comment (mlejva) [inline examples/codestral-code-interpreter-ts/README.md:30]: Add link to the file so people can copy it

File: examples/codestral-code-interpreter-ts/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## Run the program:
+
+    npm run start
```
Comment (mlejva) [inline examples/codestral-code-interpreter-ts/README.md:35]: Use the formatting with backticks not 4 spaces

File: examples/codestral-code-interpreter-ts/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# AI Code Execution with Mistral's Codestral
+
+This project tests capabilities of the new Mistral's model on data analysis tasks, using the [Code Interpreter SDK](https://github.com/e2b-dev/code-interpreter) by E2B. Codestral doesn't support using tools for code execution yet, so in this Python example, we added the code interpreting capabilities. We are going to build an AI agent that performs data analysis tasks on provided data, in a form of a csv file.
```
Comment (mlejva) [inline examples/codestral-code-interpreter-ts/README.md:3]: "Codestral doesn't support using tools for code execution yet"

should be

"odestral doesn't support using tools yet"

Comment (mlejva) [reply examples/codestral-code-interpreter-ts/README.md:3]: "We are going to build an AI agent that performs data analysis tasks on provided data, in a form of a csv file."

1. Not sure if this is an AI agent but I'll leave it up to you
2. I'd simplify it to "We are going to build an AI agent that performs data analysis on a CSV file."

File: examples/codestral-code-interpreter-ts/README.md
Comment (mlejva) [inline examples/codestral-code-interpreter-ts/README.md:1]: Add the generated image and the user prompt (not system prompt) to the readme so users know what this example does

**Comment by mlejva** [comment]
Add `.env.template` that you're referencing readme

**Comment by tizkovatereza** [comment]
All comments incorporated, thank you! 🙏

File: examples/codestral-code-interpreter-js/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# AI Code Execution with Mistral's Codestral
+
+This project tests the capabilities of the new Mistral's model on data analysis tasks, using the [Code Interpreter SDK](https://github.com/e2b-dev/code-interpreter) by E2B. Codestral doesn't support using tools yet, so in this Python example, we added the code interpreting capabilities.
+
+We are going to build an AI assistant that performs data analysis on a CSV file. It is prompted to plot the average temperature over the years in Algeria,
+
+
+# Installation
+
+First, cone the E2B Cookbook repository, and navigate to the correct directory.
```
Comment (mlejva) [inline examples/codestral-code-interpreter-js/README.md:10]: Typo - "cone". You don't need to mention it though, people know that. First step is installing dependencies

File: examples/codestral-code-interpreter-js/README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+## Install dependencies
+
+Ensure all dependencies are installed via `npm install`.
+
+
+## Set up environment variables
+
+Create a `.env` file in the project root directory and add your API keys:
+
+- Copy `.env.template`   to `.env`
```
Comment (mlejva) [inline examples/codestral-code-interpreter-js/README.md:22]: One to many spaces before "to"

File: examples/codestral-code-interpreter-js/README.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+- Get the [E2B API KEY](https://e2b.dev/docs/getting-started/api-key)
+- Get the [MISTRAL API KEY](https://console.mistral.ai/api-keys/)
+
+# Usage
+
+Ensure you have the [`city_temperature.csv`](https://github.com/e2b-dev/e2b-cookbook/blob/main/examples/codestral-code-interpreter-js/city_temperature.csv) dataset file in the project root directory.
```
Comment (mlejva) [inline examples/codestral-code-interpreter-js/README.md:28]: They will have it there if they cloned the repo. So not sure if you need to mention it

**Comment by tizkovatereza** [comment]
Fixed

**Pull request merged**